Library/Homebrew/test/lib/config.rb

```diff
@@ -11,7 +11,7 @@
 HOMEBREW_PREFIX        = Pathname.new(TEST_TMPDIR).join("prefix")
 HOMEBREW_REPOSITORY    = HOMEBREW_PREFIX
 HOMEBREW_LIBRARY       = HOMEBREW_REPOSITORY+"Library"
-HOMEBREW_LIBRARY_PATH  = File.expand_path("../../..", __FILE__)
+HOMEBREW_LIBRARY_PATH  = Pathname.new(File.expand_path("../../..", __FILE__))
```

Comment:
This is a Pathname in the other `config.rb` so making this consistent seems a no-brainer.